src/rtos/rtos-chibios.ts

```diff
@@ -596,16 +959,58 @@ export class RTOSChibiOS extends RTOSCommon.RTOSBase {
             htmlContent.css = lastHtmlInfo.css;
             return htmlContent;
         } else if (this.finalThreads.length === 0) {
-            htmlContent.html = `<p>No ${this.name} threads detected, perhaps RTOS not yet initialized or tasks yet to be created!</p>\n`;
+            htmlContent.html =
+                            `<p>No ${this.name} threads detected, perhaps RTOS not yet initialized or
+                            tasks yet to be created!</p>\n`;
             return htmlContent;
         }
 
-        const ret = this.getHTMLCommon(DisplayFieldNames, ChibiOSItems, this.finalThreads, this.timeInfo);
-        htmlContent.html = ret.html + (this.helpHtml || '');
-        htmlContent.css = ret.css;
+        this.createHmlHelp();
+
+        const htmlThreads = this.getHTMLCommon(threadDisplayFieldNames, threadTableItems, this.finalThreads, '');
+
+        const htmlGlobalInfo = this.getHTMLDataGrid(globalInfoCols,
+                                                    this.globalInfo,
+                                                    [{ name: 'id', value: 'global' },
+                                                     { name: 'aria-label', value: 'Global Variables' },
+                                                     { name: 'grid-template-columns', value: '30% 70%' }]);
+
+        const htmlVirtualTimersInfo = this.getHTMLDataGrid(virtualTimersCols,
+                                                           this.virtualTimersInfo,
+                                                           [{ name: 'id', value: 'timers' },
+                                                            { name: 'aria-label', value: 'Virtual Timers' }]);
+
+        const htmlStatistics = this.getHTMLDataGrid(statisticsCols,
+                                                    this.statistics,
+                                                    [{ name: 'id', value: 'statistics' },
+                                                     { name: 'aria-label', value: 'Statistics' }]);
+
+        const htmlRTOSPanels = this.getHTMLPanels([{ title: 'GLOBAL' },
+                                                   { title: `THREADS
```

Comment:
I would like to see the threads as the default (first) entry.

I think this is the first thing a user wants to see in the first place to get an overview of the OS state.

@haneefdm Any opinion regarding this?